grails-app/controllers/org/pih/warehouse/core/ErrorsController.groovy

```diff
@@ -96,12 +96,39 @@ class ErrorsController {
         if (RequestUtil.isAjax(request)) {
             response.status = 400
             BeanPropertyBindingResult errors = request?.exception?.cause?.errors
+            def validationErrors = errors?.fieldErrors?.collect {
```

Comment:
I kind of understand what's happening here. But there's something I don't like about doing this block of code. Is there another way to achieve this without the hardcoded Location logic. And why is the marshaller being invoked on a validation error? Do we really send back the entire object on validation errors? 

@pmuchowski Could you remove the logic and record a video of the behavior you're seeing?

Reply:
@jmiranda we're not using the validation error anywhere on the frontend, we're just using the error messages, do you think it's fine to just remove it from the json?

Reply:
That would probably be ok. Although there are some cases where we'd like the client to know what the validation errors are. But if you're saying that we're already getting all validation errors as processed through the message service and not the raw components of validation errors, then yes I think we can remove the unprocessed validation errors.

Reply:
@jmiranda yes, we're using the message service to process all of the validation errors, so all of the validation errors are already displayed on the frontend.
In this case I will just remove the plain object, because it's not used anyway.